pkg/object/pipeline/pipeline.go

```diff
@@ -69,12 +69,16 @@ type (
 
 	// FlowNode describes one node of the pipeline flow.
 	FlowNode struct {
-		Filter           string            `yaml:"filter" jsonschema:"required,format=urlname"`
-		DefaultRequestID string            `yaml:"defaultRequestID" jsonschema:"defaultRequestID,omitempty"`
-		TargetRequestID  string            `yaml:"targetRequestID" jsonschema:"targetRequestID,omitempty"`
-		TargetResponseID string            `yaml:"targetResponseID" jsonschema:"targetResponseID,omitempty"`
-		JumpIf           map[string]string `yaml:"jumpIf" jsonschema:"omitempty"`
-		filter           filters.Filter
+		Filter string `yaml:"filter" jsonschema:"required,format=urlname"`
+		// Note, the JSON tag of `DefaultRequest` is 'useRequest`, which is
+		// different from its name, because, from the aspect of code,
+		// `DefaultRequest` is to set the default request for the filter, but
+		// from the view of users, `useRequest` is much easier to understand.
+		DefaultRequest string            `yaml:"useRequest" jsonschema:"useRequest,omitempty"`
```

Comment:
is change `DefaultRequest` to `UseRequest ` better?

Reply:
from the developer view, `DefaultRequest` describes its usage better.